graphistry/layout/fa2.py

```diff
@@ -138,11 +140,28 @@ def fa2_layout(
 
         if engine_concrete == Engine.PANDAS:
             logger.warning("Pandas engine detected. FA2 falling back to igraph fr")
-            g_connected_layout = g_connected.layout_igraph(
-                'fr',
-                directed=False,
-                params=fa2_params if fa2_params is not None else GRAPHISTRY_FR_PARAMS
-            )
+            try:
```

Comment:
why did we add this? seems incorrect, they asked for fa2 and we're making it do something else rather than exn